GUI/src/cz/fidentis/gui/NavigatorTopComponent.java

```diff
@@ -385,6 +389,10 @@ public void processSelection(TreePath path) {
                         ModelLoader loader = new ModelLoader();
                         Model model = loader.loadModel(file, true, true);
                         listener.setModels(model);
+                        if(batchComparison.getRegistrationMethod() == RegistrationMethod.PROCRUSTES)   { //in case procrustes if picked do this

```

Comment:
Toto je tam kvoli tomu aby sa zobrazili feature points modelu? ak ano, myslim ze toto nie je uplne spravny postup lebo feature pointy k modelu mozu byt aj v inych modoch atd.

Lepsie by bolo vzdy sa opytat ci k modelu existuju FP a ak ano tak ich zobrazit (to mala robit podmienka ktora tam bola predtym - `comparison.getPreregiteredModels() != null`)

To iste plati ku kodu par riadkov nizsie